# This -Werror will cause the build to fail because of a warning about using the intel compiler
# Ward Poelmans <ward.poelmans@ugent.be>
diff -ur mariadb-10.1.13.orig/storage/tokudb/PerconaFT/cmake_modules/TokuSetupCompiler.cmake mariadb-10.1.13/storage/tokudb/PerconaFT/cmake_modules/TokuSetupCompiler.cmake
--- mariadb-10.1.13.orig/storage/tokudb/PerconaFT/cmake_modules/TokuSetupCompiler.cmake	2016-03-24 10:12:12.000000000 +0100
+++ mariadb-10.1.13/storage/tokudb/PerconaFT/cmake_modules/TokuSetupCompiler.cmake	2016-04-20 14:29:31.019940551 +0200
@@ -156,8 +156,8 @@
 endif ()
 
 ## always want these
-set(CMAKE_C_FLAGS "-Wall -Werror ${CMAKE_C_FLAGS}")
-set(CMAKE_CXX_FLAGS "-Wall -Werror ${CMAKE_CXX_FLAGS}")
+set(CMAKE_C_FLAGS "-Wall ${CMAKE_C_FLAGS}")
+set(CMAKE_CXX_FLAGS "-Wall ${CMAKE_CXX_FLAGS}")
 
 ## need to set -stdlib=libc++ to get real c++11 support on darwin
 if (APPLE)
